src/commands/kick.js

```diff
@@ -43,19 +55,19 @@ async function kickHandler(interaction) {
 				value: user.id
 			},
 			{
-				name: 'Executor',
+				name: 'Moderator',
 				value: `<@${executor.id}>`
 			},
 			{
-				name: 'Executor User ID',
+				name: 'Moderator User ID',
 				value: executor.id
 			},
 			{
 				name: 'Reason',
 				value: reason
 			},
 			{
-				name: 'From bot /kick command',
+				name: 'From Bot',
```

Comment:
It's already clear this is from the bot, that's the only way these messages get sent. The point of this naming convention was to note if the kick was specifically from calling the `/kick` command, or if it was from the result of multiple warnings. Both of those actions come from the bot, this is used to know WHICH action it was